### ISSUE TITLE
Feature request: allow specification of device classes for health sensors in the same way as for presence sensors

### ISSUE THREAD

**Issue by cerebrate** [jseidl/hass-magic_areas#141]
At the moment, as I understand it, health sensors include classes problem, smoke, moisture, safety, and gas.

Trouble is, for me, all the sensors of type problem are tamper entities attached to some other sensor, which seems to me to be a different class of issue. I get an aggregate sensor of Area Problem ([room]), which I treat as my "problem with some sensor in here, go fix it" warning, but the same issues show up in my Area Health ([room]) sensor, which I want to treat more urgently, as smoke, flood, gas, and suchlike obviously need to be treated. I can't get the tamper sensors out of health without excluding the entities entirely, at which point I don't get the Problem aggregate either.

**Comment by jseidl** [comment]
Good point. This should be relatively straightforward to implement, added to the queue
